### PR TITLE
Update CONVENTIONS.lagda.md

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Add instructions for adding links and misc. small improvements.

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
Also, I think it is time to decide a convention for adding references to articles and other research material.

**Comment by fredrik-bakke** [comment]
Actually, should other files be referenced as `.md`'s or `.html`'s?

**Comment by EgbertRijke** [comment]
I think other files should be referenced as `.md`

**Comment by fredrik-bakke** [comment]
Alright, I will create a second PR to fix that.

File: src/CONVENTIONS.lagda.md [hunk 1]
```diff
@@ -6,17 +6,20 @@
 3. Files that are part of the formalisation should be in literate agda using markdown. They should have the file extension `.lagda.md`.
 4. Every file should begin with a header in the following format
 ```md
-# [The title of the file]
+# (The title of the file)
```
Comment (EgbertRijke) [inline src/CONVENTIONS.lagda.md:9]: Perhaps even without the parentheses

**Pull request merged**